apps/ex_nvr_web/lib/ex_nvr_web/live/device_live.html.heex

```diff
@@ -51,6 +51,21 @@
         </p>
         <div class="flex flex-col">
           <.inputs_for :let={config} field={@device_form[:stream_config]}>
+            <.input
+              :if={@device_type == "ip"}
+              field={@device_form[:vendor]}
+              id="device_vendor"
+              type="select"
+              options={
+                [
+                  {"Hikvision", "hikvision"}, 
+                  {"Milesight Technology Co.,Ltd", "milesight"},
+                  {"Axis", "axis"}
```

Comment:
This field should be `AXIS` and add an empty option if the camera type is not in this list.
And I think this is not the place to put this input, you're inside the stream config field.